### PR TITLE
Reduce heap size to 2KB

### PR DESCRIPTION
**Pull request by randywu763**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: SAM_IoT_Hub.md [hunk 1]
```diff
@@ -114,70 +122,64 @@ can assign to &gt; leave the rest as their existing defaults &gt; hit
...
 
-    - Connect the board to PC, then make sure `CURIOSITY` device shows up as a disk drive in a `File Explorer` window
+    - Open a serial terminal (e.g. PuTTY, TeraTerm, etc.) and connect to the COM port corresponding to your board at 9600 baud (e.g. open PuTTY Configuration window &gt; choose `session` &gt; choose `Serial`&gt; Enter the right COMx port). You can find the COM info by opening your PC’s `Device Manager` &gt; expand `Ports(COM & LPT)` &gt; take note of `Curiosity Virtual COM Port` 
 
-    - Right-click the project `AzureIotPnPDps` &gt; select `Properties` &gt; select "SAMD21-IoT WG-SN" for `Connected Hardware Tool` &gt; select the latest `SAMD21_DFP` version &gt; select the latest XC32 version for `Compiler Toolchain`. If any changes were made in the project properties window, the `Apply` button should become enabled.  Make sure to hit the `Apply` button before hitting `OK`
+        <img src=".//media/image27.png">
 
-5. Build the project and set up a Command Line Interface (CLI) to the board:
+    - In your `Terminal Options`, enable `Local Echo` to allow your keystrokes to be displayed in the terminal window
```
Comment (danewalton) [inline SAM_IoT_Hub.md:137]: this seems to conflict with what is said on the next line?

File: SAM_IoT_Central.md [hunk 2]
```diff
@@ -6,6 +6,16 @@
 
 The web UI lets you quickly connect devices, monitor device conditions, create rules, and manage millions of devices and their data throughout their life cycle. Furthermore, it enables you to act on device insights by extending IoT intelligence into line-of-business applications.
 
+[IoT Plug and Play](https://docs.microsoft.com/en-us/azure/iot-develop/overview-iot-plug-and-play) enables solution builders to integrate IoT devices with their solutions without any manual configuration. At the core of IoT Plug and Play, is a device model that a device uses to advertise its capabilities to an IoT Plug and Play-enabled application. This model is structured as a set of elements that define:
+
+- `Properties` that represent the read-only or writable state of a device or other entity. For example, a device serial number may be a read-only property and a target temperature on a thermostat may be a writable property
+
+- `Telemetry` that's the data emitted by a device, whether the data is a regular stream of sensor readings, an occasional error, or an information message
```
Comment (danewalton) [inline SAM_IoT_Central.md:13]:
```suggestion
- `Telemetry` which is the data emitted by a device, whether the data is a regular stream of sensor readings, an occasional error, or an information message
```

**Review by danewalton — APPROVED** [review]

File: SAM_IoT_Hub.md (hunk 1 above)
Comment (randywu763) [reply SAM_IoT_Hub.md]: Yes good catch - removed this line (no need to enable local echo for the SAM-IoT; that only applies to the PIC-IoT)

File: SAM_IoT_Central.md [hunk 3]
```diff
@@ -16,57 +26,58 @@ The web UI lets you quickly connect devices, monitor device conditions, create r
...
+3. Set up a Command Line Interface (CLI) to the board
 
     - Open a serial terminal (e.g. PuTTY, TeraTerm, etc.) and connect to the COM port corresponding to your board at 9600 baud (e.g. open PuTTY Configuration window &gt; choose `session` &gt; choose `Serial`&gt; Enter the right COMx port). You can find the COM info by opening your PC’s `Device Manager` &gt; expand `Ports(COM & LPT)` &gt; take note of `Curiosity Virtual COM Port` 
 
-        <img src=".//media/image27.png" style="width:3in;height:3in" alt="A screenshot of a cell phone Description automatically generated" />
+        <img src=".//media/image27.png">
 
-    - Right-click the `AzureIotPnPDps` project and select `Make and Program Device`.  This operation will automatically clean and build the project before attempting to program the target device. After the `BUILD SUCCESSFUL` message appears in the Output window, the application HEX file will be programmed onto the SAM-IoT board. Once programming has finished, the board will automatically reset and start running its application code.
+    - In your `Terminal Options`, enable `Local Echo` to allow your keystrokes to be displayed in the terminal window
```
Comment (danewalton) [inline SAM_IoT_Central.md:41]: I think it's still here

**Pull request merged**